### DISCUSSION TITLE
Hey I just posted submitted my profile for review today

### DISCUSSION THREAD

**Discussion by rtpa25** [github/india#689] [Students]
And the return email states that it takes them around 48-72 hours to verify a profile and the last date to apply to any organization is tomorrow, will I able to get my profile verified by tomorrow 
Would really appreciate any form of assistance

**Comment by rungta-ayush** [comment]
+1

**Comment by jaygoyanar** [comment]
@rtpa25, What is the status of your profile?

**Reply by rungta-ayush** [reply]
"Your profile is under review, looks like you have not filled the complete form and thats causing some delay. We request you to kindly update the information if not done."
Please help

**Comment by rtpa25** [comment]
@jaygoyanar hey my profile just got approved thanks for the assistance

**Comment by rungta-ayush** [comment]
@jaygoyanar 
Can you also please look into my profile.
My ID is : rungta-ayush
Email id: ayush.ninza@gmail.com

**Reply by rungta-ayush** [reply]
**ISSUE RESOLVED**
